readme.txt

```diff
@@ -176,6 +176,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Tweak - A new `tribe_events_attendees_csv_export_delimiter` filter was introduced to help change the delimiter when creating a CSV export of attendees. [GTRIA-508]
```

Comment:
We have to revisit the ticket ID once we define in which version we're releasing this. Normally when it goes from triage to a release, the ticket ID changes.